src/Microsoft.Dnx.Host/PathBasedAssemblyLoader.cs

```diff
@@ -23,8 +23,14 @@ public PathBasedAssemblyLoader(IAssemblyLoadContextAccessor loadContextAccessor,
 
         public Assembly Load(AssemblyName assemblyName)
         {
-            foreach (var path in _searchPaths)
+            foreach (var searchPath in _searchPaths)
```

Comment:
Can you add a comment that calls out the search paths in both cases (with and without a culture?)